docs/status.md

```diff
@@ -122,7 +122,7 @@ one of the following tracking labels.
 | round_nearest_even       | no            | revisit      | revisit        | yes             | no          |
 | rsqrt                    | yes           | yes          | yes            | yes             | no          |
 | scatter                  | no            | revisit      | no             | no              | no          |
-| select                   | no            | yes*         | yes*           | yes             | no          |
+| select                   | yes           | yes          | yes            | yes             | no          |
```

Comment:
I also see that `SelectOp::verify` allows `pred` to have a different shape from `on_true`. What's the story there? We don't generally allow implicit broadcasting in StableHLO/MHLO. Why allow it here?

Reply:
As MHLO::selectOp does support rank-0 pred, whereas the HLO pred does not, the MHLO to HLO exporter inserts broadcasts. There could be reason for MHLO to diverge from HLO, but it needs investigation to find that out.
@subhankarshah Can you do that and document the results of the investigation in PR comments?

Reply:
Looking into this

Reply:
Jax and Tensorflow depend on scalar pred with on_true and on_false as ranked tensors.

Reply:
Good find! Let's proceed with the spec as is then and sort this out later in #442.